pkg/components/prometheus-operator/template.go

```diff
@@ -88,20 +91,28 @@ kubeEtcd:
   enabled: {{.Monitor.Etcd}}
 prometheus-node-exporter:
   service: {}
-{{ if (or .PrometheusOperatorNodeSelector .DisableWebhooks) }}
+{{ if (or .Operator.NodeSelector .DisableWebhooks .Operator.Tolerations ) }}
 prometheusOperator:
   {{- if .DisableWebhooks }}
   tlsProxy:
     enabled: false
   admissionWebhooks:
     enabled: false
   {{- end }}
-  {{- if .PrometheusOperatorNodeSelector }}
+  {{- if .Operator.AdmissionWebhookTolerations }}
+  admissionWebhooks:
```

Comment:
If both webhook is disabled and tolerations for it are set, there will be duplicated key in YAML.